### PR TITLE
[Merge after release] `main` to `develop`

### PR DESCRIPTION
**Pull request by MarinTolic**
### 🎯 Goal

Main needs to be merged to develop after `v5.12.0` was released.

### 🧪 Testing

After checking the code, run the apps and perform various general chat actions to make sure everything has been merged fine.

#### General
- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [ ] Assigned a person / code owner group (required)
- [ ] Thread with the PR link started in a respective Slack channel (#android-chat-core or #android-chat-ui) (required)
- [ ] PR targets the `develop` branch
- [ ] PR is linked to the GitHub issue it resolves

#### Code & documentation
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (KDocs, docusaurus, tutorial)

### ☑️Reviewer Checklist
- [ ] UI Components sample runs & works
- [ ] Compose sample runs & works
- [ ] UI Changes correct (before & after images)
- [ ] Bugs validated (bugfixes)
- [ ] New feature tested and works
- [ ] Release notes and docs clearly describe changes
- [ ] All code we touched has new or updated KDocs

### 🎉 GIF

_Please provide a suitable gif that describes your work on this pull request_

### CODE REVIEW

File: stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api/models/identifier/Identifiers.kt [hunk 1]
```diff
@@ -224,6 +224,19 @@ internal fun HideChannelIdentifier(
     return result
 }
 
+/**
+ * Identifier for a [ChatClient.getMessage] call.
+ */
+@Suppress("FunctionName", "MagicNumber")
```
Comment (MarinTolic) [inline stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api/models/identifier/Identifiers.kt:230]: Change made when adding OP to `ChatClient.getMessage()` in `v5`.

It's non-conflicting with the one made in the `v6` PR seen here: https://github.com/GetStream/stream-chat-android/pull/4635/files#diff-ef661b55135a2b5ebd91e17fe89f833aaf5dd222d11d1e917f5ab17cc9878f0fR227

File: stream-chat-android-state/src/main/java/io/getstream/chat/android/state/plugin/logic/internal/LogicRegistry.kt [hunk 2]
```diff
@@ -128,11 +128,29 @@ internal class LogicRegistry internal constructor(
         }
     }
 
+    /**
```
Comment (MarinTolic) [inline stream-chat-android-state/src/main/java/io/getstream/chat/android/state/plugin/logic/internal/LogicRegistry.kt:131]: Also non conflicting with the `v6` version of this feature seen in the following pr: https://github.com/GetStream/stream-chat-android/pull/4639/files#diff-bdd7ddb84df9b26d6bb6f97be0c1f459d03af78461571cb22937590ac4293eb5

File: stream-chat-android-state/src/main/java/io/getstream/chat/android/state/plugin/state/internal/ChatClientStateCalls.kt [hunk 3]
```diff
@@ -81,4 +81,23 @@ internal class ChatClientStateCalls(
         chatClient.getReplies(messageId, messageLimit).launch(scope)
         return state.thread(messageId)
     }
+
```
Comment (MarinTolic) [inline stream-chat-android-state/src/main/java/io/getstream/chat/android/state/plugin/state/internal/ChatClientStateCalls.kt:84]: Likewise has a non-conflicting change in it's `v6` thread PN navigation PR counterpart: https://github.com/GetStream/stream-chat-android/pull/4639/files#diff-56fdba7a308122f62f337bdda5c56fe732394b3674d4459686ca3d4177ea3780

File: stream-chat-android-ui-common/src/main/kotlin/io/getstream/chat/android/ui/common/feature/messages/composer/MessageComposerController.kt [hunk 4]
```diff
@@ -323,10 +336,20 @@ public class MessageComposerController(
     /**
      * Sets up the observing operations for various composer states.
      */
+    @OptIn(FlowPreview::class)
     private fun setupComposerState() {
         input.onEach { input ->
             state.value = state.value.copy(inputValue = input)
-        }.launchIn(scope)
+
```
Comment (MarinTolic) [inline stream-chat-android-ui-common/src/main/kotlin/io/getstream/chat/android/ui/common/feature/messages/composer/MessageComposerController.kt:343]: The changes in this file are a part of the mention online member query PR

**Review by TGazica — APPROVED** [review]

**Review by adasiewiczr — APPROVED** [review]

**Pull request merged**